lib/regex.js

```diff
@@ -1,34 +1,22 @@
 'use strict';
-var forEach = require('lodash').forEach;
 
+function join(array, joiner) {
+  return array
+    .map(function(val) {
+      return (val || '').trim();
```

Comment:
You had an `if (val)` in the old version, so I added it just incase you were checking against the user providing null or something.